test/e2e/nfd_gc_test.go

```diff
@@ -38,7 +38,7 @@ import (
 )
 
 // Actual test suite
-var _ = SIGDescribe("NFD GC", func() {
+var _ = NFDDescribe(Label("e2e-test", "nfd-gc"), func() {
```

Comment:
I think we could drop the `e2e-test` label. Or then specify it in the NFDDescribe helper

```suggestion
var _ = NFDDescribe(Label("nfd-gc"), func() {
```